index.js

```diff
@@ -14,14 +14,25 @@ var builder = require('./lib/builder');
 
 var path = require('path');
 
-/* Very basic, null-friendly, shallow clone for object attributes only */
-function clone(obj) {
-  obj = obj || {};
-  var copy = {};
-  for (var key in obj) {
-    copy[key] = obj[key];
-  }
-  return copy;
+function processOpts(opts_, outFile) {
+  var keys = [
+    'config',
+    'lowResSourceMaps',
+    'minify',
+    'normalize',
+    'outFile',
+    'runtime',
+    'sourceMaps',
+    'sourceMapContents'
+  ];
+  var opts = {};
+  keys.forEach(function(key) {
+    opts[key] = opts_ && opts_[key];
+  });
```

Comment:
Current behaviour has every default or falsey. Assume your example was just to demonstrate, and you're not wanting to enforce new defaults?